chromadb/server/fastapi/__init__.py

```diff
@@ -744,6 +745,34 @@ async def delete_collection(
             limiter=self._capacity_limiter,
         )
 
+    @staticmethod
+    def _generate_ids_when_not_present(
+        ids: Optional[List[str]],
+        documents: Optional[List[Optional[str]]],
```

Comment:
nit: why do you need to pass docs/urs/embeddings into this, it makes more sense to me that this would take the value N that it needs